custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -35,13 +36,40 @@ def allowed_to_forward(self, episode_case):
             not episode_case_properties.get('nikshay_id', False) and
 
             # Episode pending registration flips from yes to no
-            case_properties_changed(episode_case, ['episode_pending_registration']) and
-            episode_case_properties.get('episode_pending_registration', 'yes') == 'no'
+            case_properties_changed(episode_case, [EPISODE_PENDING_REGISTRATION]) and
+            episode_case_properties.get(EPISODE_PENDING_REGISTRATION, 'yes') == 'no'
+        )
+
+
+class NikshayTreatmentOutcomeRepeater(CaseRepeater):
+    class Meta(object):
+        app_label = 'repeaters'
+
+    friendly_name = _("Forward Treatment Outcomes to Nikshay (episode case type)")
+
+    @classmethod
+    def available_for_domain(cls, domain):
+        return NIKSHAY_INTEGRATION.enabled(domain)
+
+    @classmethod
+    def get_custom_url(cls, domain):
+        from custom.enikshay.integrations.nikshay.views import NikshayTreatmentOutcomesView
+        return reverse(NikshayTreatmentOutcomesView.urlname, args=[domain])
+
+    def allowed_to_forward(self, episode_case):
+        allowed_case_types_and_users = self._allowed_case_type(episode_case) and self._allowed_user(episode_case)
+        episode_case_properties = episode_case.dynamic_case_properties()
+        return allowed_case_types_and_users and (
+            not episode_case_properties.get('nikshay_registered', 'false') == 'true' and
+            not episode_case_properties.get('nikshay_id', False) and
+            not episode_case_properties.get('treatment_outcome_nikshay_registered', False) and
```

Comment:
Is the `treatment_outcome_nikshay_registered` case property a boolean? If so I think this should be
```python
episode_case_properties.get('treatment_outcome_nikshay_registered', False) == 'true'
```